code/modules/power/apc.dm

```diff
@@ -499,12 +499,12 @@
 						return
 					else if (obj_flags & EMAGGED)
 						obj_flags &= ~EMAGGED
-						user.visible_message(span_notice("[user.name] discards an emagged power control board from [src.name]!"),\
-							span_notice("You discard the emagged power control board."))
+						user.visible_message(span_notice("[user.name] discards a fried power control board from [src.name]!"),\ 
+							span_notice("You discard the fried power control board.")) // SKYRAT EDIT -- PLAUSIBLE DENIABILITY -- (emagged/hacked APCs don't reveal their cause)
 						return
 					else if (malfhack)
```

Comment:
` code/modules/power/apc.dm:505:error:  : undefined var`

What the fuck do you want from me dream maker